### PR TITLE
b-5-4-get-all-orders(frontend)

### PR DESCRIPTION
**Pull request by ValentynaBondarenko**
1. Create all-orders.html;
2. Changed folder place user.js

### CODE REVIEW

File: src/main/webapp/assets/js/page/all-orders.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function showHideFilterTable(element) {
+  var srcElement = document.getElementById(element);
```
Comment (Brutt) [inline src/main/webapp/assets/js/page/all-orders.js]: change var to let

File: src/main/webapp/assets/js/page/all-orders.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	 if (srcElement != null) {
+		if (srcElement.style.display == "block") {
+			srcElement.style.display = 'none';
+		} else {
+			srcElement.style.display = 'block';
+		}
+	return false;
```
Comment (Brutt) [inline src/main/webapp/assets/js/page/all-orders.js]: return is not needed

File: src/main/webapp/WEB-INF/templates/all-orders.html [hunk 3]
```diff
@@ -4,78 +4,108 @@
...
-            <label for="inputState">State</label>
-            <select id="inputState" name="filterOrderStatus" class="form-control">
-                <option selected></option>
-                <option>New</option>
-                <option>View and Select</option>
-                <option>Selected</option>
-                <option>Ready</option>
-            </select>
+
+    <div class="order-board" id="filterTable" style="display:none">
```
Comment (Brutt) [inline src/main/webapp/WEB-INF/templates/all-orders.html]: maybe it is better to create a class "hide" and toggle it if click Filter button

File: src/main/webapp/assets/css/all-orders.css [hunk 4]
```diff
@@ -0,0 +1,142 @@
+.orders{
+	max-width: 1440px;
+	margin: 0, auto;
```
Comment (Rooman) [inline src/main/webapp/assets/css/all-orders.css]: :man_facepalming:

File: src/main/webapp/assets/css/all-orders.css [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+.table-column-name {
+	border-radius: 10px ;
+	background: rgba(250, 225, 222, 0.8);
+	font-weight: 300;
+	font-size: 48px;
+	line-height: 48px;
+	text-align: center;
+}
+.table-column-name {
```
Comment (Rooman) [inline src/main/webapp/assets/css/all-orders.css]: :man_facepalming:

File: src/main/webapp/assets/css/all-orders.css [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+.table-column-content {
+	font-weight: 300;
+	font-size: 36px;
+	line-height: 36px;
+	text-align: center;
+}
+
+td,
```
Comment (Rooman) [inline src/main/webapp/assets/css/all-orders.css]: это бэм, мы вешаем на классы

File: src/main/webapp/assets/css/common.css [hunk 7]
```diff
@@ -83,6 +82,91 @@ body {
...
+}
+
+.breadcrumb {
+	margin: 0;
+	padding: 0;
+	background-color: white;
+}
+
+.bread__crumb-arrow {
+	margin-top: 1.5625rem;
```
Comment (Rooman) [inline src/main/webapp/assets/css/common.css]: точность пугает

File: src/main/webapp/assets/js/page/all-orders.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+function showHideFilterTable(element) {
+  let srcElement = document.getElementById(element);
```
Comment (Rooman) [inline src/main/webapp/assets/js/page/all-orders.js]: const

**Pull request merged**